src/devices/shelly.ts

```diff
@@ -45,6 +48,313 @@ const shellyModernExtend = {
             }),
         ];
     },
+    shellyRPCSetup(features: string[] = []): ModernExtend {
+        // Set helper variables
+        const shellyRPCBugFixed = false; // For firmware 20250819-150402/ga0def2d
+
+        const featureDev = features.includes("Dev");
+        const featurePowerstripUI = features.includes("PowerstripUI");
+
+        // Generic helper functions
+        const sleepMsec = async (s: number) => {
```

Comment:
Please use this sleep function: https://github.com/Koenkk/zigbee-herdsman-converters/blob/f3aa06c9ff5eb10fd43b6c12b1baf7351c0956a5/src/lib/utils.ts#L355